.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,52 +1,7 @@
-
-1. Please write a description in this text box of the changes that are being
-   made.
-
-2. Please ensure that you have written units tests for the changes made/features
-   added.
-
-3. There are CI checks in place to enforce that committed code follows our style
-   and syntax standards. Please see our contribution guide in `CONTRIBUTING.MD`
-   in the project root for more information about the checks we perform and how
-   you can run them locally.
-
-4. If you are closing an issue please use one of the automatic closing words as
-   noted here: https://help.github.com/articles/closing-issues-using-keywords/
-
-5. If your pull request is not ready for review but you want to make use of the
-   continuous integration testing facilities please mark your pull request as Draft.
-   https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/changing-the-stage-of-a-pull-request#converting-a-pull-request-to-a-draft
-
-6. If your pull request is ready to be reviewed without requiring additional
-   work on top of it, then remove it from "Draft" and make it "Ready for Review".
-   https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/changing-the-stage-of-a-pull-request#marking-a-pull-request-as-ready-for-review
-
-   If assistance is required to complete the functionality, for example when the
-   C/C++ code of a feature is complete but Python bindings are still required,
-   then add the label `help wanted` so that others can triage and assist.
-   The additional changes then can be implemented on top of the same PR.
-   If the assistance is done by members of the rapidsAI team, then no
-   additional actions are required by the creator of the original PR for this,
-   otherwise the original author of the PR needs to give permission to the
-   person(s) assisting to commit to their personal fork of the project. If that
-   doesn't happen then a new PR based on the code of the original PR can be
-   opened by the person assisting, which then will be the PR that will be
-   merged.
-
-7. Once all work has been done and review has taken place please do not add
-   features or make changes out of the scope of those requested by the reviewer
-   (doing this just add delays as already reviewed code ends up having to be
-   re-reviewed/it is hard to tell what is new etc!). Further, please do not
-   rebase your branch on the target branch, force push, or rewrite history.
-   Doing any of these causes the context of any comments made by reviewers to be lost.
-   If conflicts occur against the target branch they should be resolved by
-   merging the target branch into the branch used for making the pull request.
-
-Many thanks in advance for your cooperation!
-
--->
+## Checklist
+- [ ] I am familiar with the [Contributing Guidelines](https://github.com/rapidsai/cudf/blob/HEAD/CONTRIBUTING.md).
+- [ ] New or existing tests cover these changes.
+- [ ] The documentation is up to date with these changes.
```

Comment:
Two additional things came to mind. Feel free to push back on these ideas.

```suggestion
## Checklist
- [ ] I am familiar with the [Contributing Guidelines](https://github.com/rapidsai/cudf/blob/HEAD/CONTRIBUTING.md).
- [ ] New or existing tests cover these changes.
- [ ] New or existing benchmarks cover these changes.
- [ ] The documentation is up to date with these changes.
- This change is:
   - [ ] Breaking
   - [ ] Non-breaking
```

Reply:
Benchmarks I can support -- with the caveat that C++ has the best benchmarking at present, and the Python is somewhat outdated.

Breaking/non-breaking is already handled by labels, and the CI requires us to label one or the other. I don't think it is necessary to input that information twice.

Reply:
IMO tests are going to be much more frequent additions than benchmarks (the latter is typically for new features)

Reply:
I would strongly discourage duplicating information in the template and labels. The overhead of every checkbox / button adds up. However, I do agree that our categories are hard to locate. This is the official reference, but I don't think it's linked anywhere in the cuDF repository: https://docs.rapids.ai/resources/label-checker/#summary

It would be helpful if the output message said "Missing category label (bug, doc, feature request, improvement)".

Though on second thought, why is "feature request" a supported pull request category? Sounds like it should only apply to issues...

Reply:
Adding to this, another idea is to add "category" to description text of the label. so that if I type "category", it will list only labels with "category" text in their description, and I can choose them easily.

Reply:
I like this idea. `category: doc` is longer, but it's a lot easier to distinguish the special category labels.

Reply:
I am not suggesting to include "category" in label itself (also a good idea). Just in description text of the label. That is enough for search to filter "category" when we type "category".

Reply:
Aha! The description would be a good compromise because it doesn’t require changing bot logic.

Reply:
could @rapidsai/admins or @rapidsai/cudf-admin update this? 
add "category" to the description text of the following labels: bug, doc, feature request, improvement.
This allows easily identifying the category labels during PR creation, by simply typing "category" in label box.